integration/deploy_test.go

```diff
@@ -26,8 +26,8 @@ import (
 )
 
 func TestBuildDeploy(t *testing.T) {
-	if testing.Short() || RunOnGCP() {
-		t.Skip("skipping kind integration test")
+	if testing.Short() || !RunOnGCP() {
```

Comment:
Should we fix something in `skaffold deploy` with kind, instead?

Reply:
skaffold deploy needs a registry to pull the images from, but I don't want to introduce credentials - as that causes all sorts of issues in travis (contributors can't run these tests, only maintainers). We can run a registry inside kind and that is an interesting use case that we should work out as well, but it will be a bit painful to setup, if possible at all today - how would we "tag" images that use the same registry address from inside the cluster and outside of it? Typically outside would be `localhost:xyz/image`, while from within `registry/image`. I'm actually going to open an issue for this. But not sure we want to tackle this in this PR. 
A third option could be to spin up a local registry on the node - that is possible but feels finicky as well, again, not sure how the cluster will be able to access it. 
A fourth option could be running a public insecure registry :) 
A fifth could be having an open credentials situation.
I'm not a fan of 4/5 for security/abuse protection reasons. 
Let me know if I missed something.

Reply:
Another group of ideas could be to load into kind from skaffold build and enable that functionality - again a bit bigger in scope than what I wanted to achieve here.